ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/pkg/PackageClient.scala

```diff
@@ -10,16 +10,16 @@ import com.daml.ledger.api.v1.package_service.PackageServiceGrpc.PackageServiceS
 import scala.concurrent.Future
 
 class PackageClient(val ledgerId: LedgerId, service: PackageServiceStub) {
-  private val it = new withoutledgerid.PackageClient(service)
+  private[daml] val it = new withoutledgerid.PackageClient(service)
```

Comment:
scared of that 👻

Reply:
That stuff needs adjustment, I'm not super happy right now :sweat:

Reply:
but that is for another day